package.json

```diff
@@ -3,15 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.0-placeholder-for-preset-env.2",
+    "@reduxjs/toolkit": "^1.9.5",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-icons": "^4.9.0",
     "react-redux": "^8.0.7",
-    "react-scripts": "^5.0.1",
+    "react-scripts": "^2.1.3",
```

Comment:
remove ^ from the version number, keeping ^ would automatically update the latest minor available version which may cause inconsistencies.

Reply:
done